docs/streams/developer-guide.html

```diff
@@ -0,0 +1,3045 @@
+<!--
```

Comment:
This file should not be included as it has been removed and replaced by the smaller sections.